packages/testcontainers/src/MetaChainContainer.ts

```diff
@@ -36,19 +40,34 @@ export class MetaChainContainer extends GenericContainer {
     ];
   }
 
-  public async start(configModifiers = {}): Promise<StartedMetaChainContainer> {
+  protected exposedPorts: PortWithOptionalBinding[];
+  protected networkMode?: string;
+  protected startCmds?: string[];
+  protected containerStartupTimeout: number = 120_000;
+
+  public setExposedPorts(...ports: PortWithOptionalBinding[]): this {
+    this.exposedPorts = ports;
+    return this;
+  }
+  public setNetworkMode(networkMode: string): this {
+    this.networkMode = networkMode;
+    return this;
+  }
+  public setCmd(cmds: string[]): this {
+    this.startCmds = cmds;
+    return this;
+  }
+  public setStartupTimeout(timeout: number): this {
+    this.containerStartupTimeout = timeout;
+    return this;
+  }
+  public async start(): Promise<StartedMetaChainContainer> {
     const network = await new Network().start();
```

Comment:
remove this too, it's not fluent

Reply:
How should the `start` function look like?

something like this?
```javascript
public async start(): this { }
```

Reply:
This being: `const network = await new Network().start();` since this should be injected. Network config shouldn't be set internally where it's hidden from consumer ability to configure.